learning_journal/learning_journal/routes.py

```diff
@@ -2,6 +2,6 @@ def includeme(config):
     """ This function adds routes to Pyramid's Configurator """
     config.add_static_view('static', 'static', cache_max_age=3600)
```

Comment:
If you're adding your static view in the `__init__.py` you don't need to also do it here.